scripts/push-report.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euo pipefail
+
+# This script appends code complexity reports over time. Given the amount of files, report files
+# are now stored on Azure Blob Storage. It's the source of truth - we download the current report,
+# run complexity analysis again, then push everything back to blob.
+rm -rf report
+mkdir -p report
+
+#NOTE: be sure to set AZURE_STORAGE_ACCOUNT and AZURE_STORAGE_KEY environment variables
+az storage blob download-batch -d report -s '$web'
```

Comment:
where is `$web` set?

Reply:
It's not a variable, which is why I'm using single quotes... when you enable static web site on Azure blob storage, it creates a container called `$web` by default. That's the container used to host all the web files. 

Not sure why they elected to use `$` to indicate that it's 'special' :man_shrugging:  If you're not careful, can definitely cause some problems in command line environments. The offering isn't flexible here; I can't find a way to specify another container name.